src/main/kotlin/org/taktik/icure/asyncdao/impl/CodeDAOImpl.kt

```diff
@@ -379,6 +379,29 @@ class CodeDAOImpl(
 		)
 	}
 
+	override fun listCodeIdsByTypeCodeVersionInterval(startType: String?, startCode: String?, startVersion: String?, endType: String?, endCode: String?, endVersion: String?): Flow<String> = flow {
+		val client = couchDbDispatcher.getClient(dbInstanceUrl)
+		val from = ComplexKey.of(
+			startType ?: "\u0000",
```

Comment:
Why using \u0000 for emptyValues of the startKey and not a ComplexKey.emptyObject() as the endKey ?

Reply:
The \u0000 character comes before any other, so the interval between that should encompass all the values in the view. Changed to a constant for clarity.